src/config/networks.ts

```diff
@@ -109,8 +109,8 @@ const makeNetworkList = () => {
       },
     },
     namespace: '09573a3526818a8ecd6eb92f60f1175d',
-    subscanEndpoint: 'http://127.0.0.1:4399',
-    subscanUrl: 'https://creditcoin3-testnet.subscan.io/',
+    subscanEndpoint: 'https://creditcoin3-testnet.api.subscan.io',
+    subscanUrl: 'https://subscan-cc3-testnet.creditcoin.network',
```

Comment:
This is the block explorer url, right?
In that case it should be: https://creditcoin3-testnet.subscan.io